integration/combination/test_function_with_cloudwatch_log.py

```diff
@@ -0,0 +1,19 @@
+from integration.helpers.base_test import BaseTest
+
+
+class TestFunctionWithCloudWatchLog(BaseTest):
+    def test_function_with_cloudwatch_log(self):
+        self.create_and_verify_stack("combination/function_with_cloudwatch_log")
+
+        cloudwatch_log_group_name = self.get_physical_id_by_type("AWS::Logs::LogGroup")
+        lambda_function_endpoint = self.get_physical_id_by_type("AWS::Lambda::Function")
+        cloudwatch_log_client = self.client_provider.cloudwatch_log_client
+
+        subscription_filter_result = cloudwatch_log_client.describe_subscription_filters(
+            logGroupName=cloudwatch_log_group_name
+        )
+        subscription_filter = subscription_filter_result["subscriptionFilters"][0]
+
+        self.assertEqual(len(subscription_filter_result["subscriptionFilters"]), 1)
+        self.assertTrue(lambda_function_endpoint in subscription_filter["destinationArn"])
```

Comment:
nit: You can use `assertIn` for this (which could display a more meaningful message).
But it's fine like this, no big deal.